assets/src/block-templates/quick-links/template.js

```diff
@@ -0,0 +1,43 @@
+import metadata from './block.json';
+import mainThemeUrl from '../main-theme-url';
+
+const category = ['core/column', {}, [
+  ['core/group', {className: 'group-stretched-link'}, [
+    ['core/image', {
+      align: 'center',
+      className: 'is-style-rounded-90 force-no-lightbox force-no-caption mb-0',
+      url: `${mainThemeUrl}/images/placeholders/placeholder-90x90.jpg`
+    }],
+    ['core/spacer', {height: '16px'}],
+    ['core/heading', {
+      level: 5,
+      style: {typography: {fontSize: '1rem'}},
+      textAlign: 'center',
+      placeholder: 'Category'
+    }]
+  ]]
+]];
+
+const template = ({
+  title = '',
+  backgroundColor = 'grey-05',
+}) => ([
+  ['core/group', {
+    className: `block ${metadata.classname}`,
+    align: 'full',
+    backgroundColor
+  }, [
+    ['core/group', {className: 'container'}, [
+      ['core/spacer', {height: '24px'}],
+      ['core/heading', {level: 4, placeholder: __('Enter title', 'planet4-blocks-backend'), content: title}],
+      ['core/columns', {
+        isStackedOnMobile: false,
+        className: 'is-style-mobile-carousel'
+      }, [
+        category, category, category, category, category
```

Comment:
Thinking out loud, what do you think to do something like this  `[...Array(5).keys()].map(() => category)` where `5` represents the total of categories to render. Also you can render the category here template instead of defining as a `const`.